### PR TITLE
TESTS: qty -> str

### PR DESCRIPTION
**Pull request by NoahTheDuke**
Kinda a joke but not actually. If we can just put in the raw card title, why wrap them when it's a single?

Here you go, @Saintis. No need to learn regex.

### CODE REVIEW

**Comment by NoahTheDuke** [comment]
Shoulda tested at least once before pushing, looks like.

File: test/clj/game_test/cards/icebreakers.clj [hunk 1]
```diff
@@ -310,12 +310,12 @@
...
       (new-game (default-corp)
-                (default-runner [(qty "Faust" 1)
-                                 (qty "Sacrificial Construct" 1)
-                                 (qty "Fall Guy" 1)
-                                 (qty "Astrolabe" 1)
+                (default-runner ["Faust"
+                                 "Sacrificial Construct"
+                                 "Fall Guy"
+                                 "Astrolabe"
                                  (qty "Gordian Blade" 1 )
```
Comment (Saintis) [inline test/clj/game_test/cards/icebreakers.clj]: Missed this one

File: test/clj/game_test/cards/icebreakers.clj [hunk 2]
```diff
@@ -557,7 +557,7 @@
   ;; See #3187
   (do-game
     (new-game
-      (default-corp [(qty "Zed 1.0" 1) (qty "Zed 2.0" 3) (qty "AR-Enhanced Security" 1)])
+      (default-corp ["Zed 1.0" "Zed 2.0" 3) (qty "AR-Enhanced Security"])
```
Comment (Saintis) [inline test/clj/game_test/cards/icebreakers.clj]: Too greedy match here

File: test/clj/game_test/cards/identities.clj [hunk 3]
```diff
@@ -13,7 +13,7 @@
     (do-game
       (new-game
         (make-deck "Weyland Consortium: Builder of Nations"
-                   [(qty "PAD Campaign" 1) (qty "The Cleaners" 1) (qty "Pup" 3) (qty "Oaktown Renovation" 1)])
+                   ["PAD Campaign" "The Cleaners" "Pup" 3) (qty "Oaktown Renovation"])
```
Comment (Saintis) [inline test/clj/game_test/cards/identities.clj]: Too greedy match here

File: test/clj/game_test/cards/identities.clj [hunk 4]
```diff
@@ -244,7 +244,7 @@
   ;; Pay 4 net damage to steal.  Only 3 damage left after Chronos.  No trigger of damage prevent.
   (do-game
     (new-game (make-deck "Chronos Protocol: Selective Mind-mapping" [(qty "Obokata Protocol" 5)])
-              (default-runner [(qty "Sure Gamble" 3) (qty "Inti" 1) (qty "Feedback Filter" 1)]))
+              (default-runner ["Sure Gamble" 3) (qty "Inti" "Feedback Filter"]))
```
Comment (Saintis) [inline test/clj/game_test/cards/identities.clj]: Too greedy match

File: test/clj/game_test/cards/identities.clj [hunk 5]
```diff
@@ -260,8 +260,8 @@
   ;; Chronos Protocol - Issue #1958 also affects Chronos Protocol
   (do-game
     (new-game
-      (make-deck "Chronos Protocol: Selective Mind-mapping" [(qty "Pup" 1)])
-      (default-runner [(qty "Employee Strike" 1) (qty "Scrubbed" 3) (qty "Sure Gamble" 1)]))
+      (make-deck "Chronos Protocol: Selective Mind-mapping" ["Pup"])
+      (default-runner ["Employee Strike" "Scrubbed" 3) (qty "Sure Gamble"]))
```
Comment (Saintis) [inline test/clj/game_test/cards/identities.clj]: Too greedy match

File: test/clj/game_test/cards/identities.clj [hunk 6]
```diff
@@ -583,7 +583,7 @@
   ;; Architects of Tomorrow - prompt to rez after passing bioroid
   (do-game
     (new-game
-      (make-deck "Haas-Bioroid: Architects of Tomorrow" [(qty "Eli 1.0" 2) (qty "Pup" 1)])
+      (make-deck "Haas-Bioroid: Architects of Tomorrow" ["Eli 1.0" 2) (qty "Pup"])
```
Comment (Saintis) [inline test/clj/game_test/cards/identities.clj]: Too greedy

File: test/clj/game_test/cards/identities.clj [hunk 7]
```diff
@@ -615,8 +615,8 @@
   ;; EtF - interaction with Employee Strike
   (do-game
     (new-game
-      (make-deck "Haas-Bioroid: Engineering the Future" [(qty "Eli 1.0" 3) (qty "Paywall Implementation" 1)])
-      (default-runner [(qty "Employee Strike" 1)]))
+      (make-deck "Haas-Bioroid: Engineering the Future" ["Eli 1.0" 3) (qty "Paywall Implementation"])
```
Comment (Saintis) [inline test/clj/game_test/cards/identities.clj]: Too greedy

File: test/clj/game_test/cards/identities.clj [hunk 8]
```diff
@@ -1012,8 +1012,8 @@
 (deftest leela-upgrades
   ;; Leela Patel - upgrades returned to hand in the middle of a run do not break the run. Issue #2008.
   (do-game
-    (new-game (default-corp [(qty "Crisium Grid" 3) (qty "Project Atlas" 3) (qty "Shock!" 1)])
-              (make-deck "Leela Patel: Trained Pragmatist" [(qty "Sure Gamble" 1)]))
+    (new-game (default-corp ["Crisium Grid" 3) (qty "Project Atlas" 3) (qty "Shock!"])
```
Comment (Saintis) [inline test/clj/game_test/cards/identities.clj]: Too greedy

File: test/clj/game_test/cards/identities.clj [hunk 9]
```diff
@@ -1173,7 +1173,7 @@
   ;; New Angeles Sol - interaction with runner stealing agendas
   (do-game
     (new-game
-      (make-deck "New Angeles Sol: Your News" [(qty "Paywall Implementation" 2) (qty "Breaking News" 1)])
+      (make-deck "New Angeles Sol: Your News" ["Paywall Implementation" 2) (qty "Breaking News"])
```
Comment (Saintis) [inline test/clj/game_test/cards/identities.clj]: Too greedy

File: test/clj/game_test/cards/identities.clj [hunk 10]
```diff
@@ -1586,7 +1586,7 @@
   (do-game
     (new-game
       (make-deck "SSO Industries: Fueling Innovation"
-                 [(qty "Hortum" 2) (qty "Oaktown Renovation" 2) (qty "Braintrust" 1)])
+                 ["Hortum" 2) (qty "Oaktown Renovation" 2) (qty "Braintrust"])
```
Comment (Saintis) [inline test/clj/game_test/cards/identities.clj]: Too greedy

File: test/clj/game_test/cards/identities.clj [hunk 11]
```diff
@@ -1792,8 +1792,8 @@
   (do-game
     (new-game (default-corp [(qty "Launch Campaign" 3)])
               (make-deck "Wyvern: Chemically Enhanced"
-                         [(qty "Sure Gamble" 2) (qty "Corroder" 1)
-                          (qty "Clone Chip" 1) (qty "Easy Mark" 1)]))
+                         ["Sure Gamble" 2) (qty "Corroder"
```
Comment (Saintis) [inline test/clj/game_test/cards/identities.clj]: Too greedy

**Pull request merged**